tests/testthat/test-get_upw.R

```diff
@@ -61,20 +61,39 @@ test_that("unpaywall mining works",{
   expect_true("oa_color" %in% names(upwdf))
   # the data frame is as long as all DOIs
   expect_true(nrow(upwdf) == length(dois))
-  #add line for testing green over bronze prioritization
+  #add line for testing green over bronze prioritization once implemented
 })
 
-test_that("errors can be dealt with",{
-  doi_fails <- c(
+test_that("errors are handled correctly",{
```

Comment:
```suggestion
test_that("DOIs without UPW registration do not break the loop",{
```

Reply:
Both invalid DOIs and valid DOIs that are not included/indexed (not: registered) in UPW. 
Thinking of a short way to phrase this...

Reply:
Proposed alternative